eg/examples/strace.md

```diff
@@ -0,0 +1,54 @@
+# strace
+
+trace syscalls of a command and follow child pids
+
+    strace -f ls
+
+# Basic Usage
+
+Trace syscalls of a command:
+
+    strace <command>
+
+See the syscalls of an already-running process:
+
+    strace -p <pid>
+
+See only `network` type syscalls (and/or `file`,`stat`, `process`, or any specific syscall, comma separated)
+
+    strace -e trace=%network <command>
+
+See only system calls accessing <path>
+
+    strace -P <path> <command>
+
+Increase default preview string size to 256
+
+    strace -s 256 <command>
+
+Show timestamps
+
+    strace -t <command>
+
+Show time spent in syscalls
+
+    strace -T <command>
+
+Only show a summary of syscalls
+
+    strace --summary-only/-c
+
+# Attach to a running process to see time spent in calls
+
+    strace -f -s 256 -p <pid> -T
+
```

Comment:
Would be nice to have a brief explanation of the command below. See other examples as reference.

Reply:
Added some more information about the command.